sharding/networksharding/peerShardMapper.go

```diff
@@ -72,7 +70,7 @@ func NewPeerShardMapper(
 		fallbackPkShard:  fallbackPkShard,
 		fallbackPidShard: fallbackPidShard,
 		nodesCoordinator: nodesCoordinator,
-		epochHandler:     epochHandler,
+		epoch:            uint32(0),
```

Comment:
you should make epoch as an argument at constructor. - for start in epoch.

Reply:
ok, will instantiate with the value stored in epochHandler.Epoch()